assets/src/apps/v2/triptych.tsx

```diff
@@ -30,15 +30,21 @@ import NoData from "Components/v2/triptych/no_data";
 
 import Placeholder from "Components/v2/placeholder";
 import TrainCrowding from "Components/v2/train_crowding";
-import EvergreenContent from "Components/v2/evergreen_content";
+import EvergreenContent from "Components/v2/triptych/evergreen_content";
 
 const customizeEvergreenProps = (WrappedComponent: React.ElementType) => {
-  return (props: { asset_url: string }) => {
+  return (props: { asset_url: string; show_identifiers: boolean }) => {
     const modifiedUrl = props.asset_url.replace("assets/static/images/", "");
     const dupReadyUrl = imagePath(modifiedUrl);
 
     const isPlaying = useIsOnScreen();
-    return <WrappedComponent asset_url={dupReadyUrl} isPlaying={isPlaying} />;
+    return (
+      <WrappedComponent
+        asset_url={dupReadyUrl}
+        isPlaying={isPlaying}
+        show_identifiers={props.show_identifiers}
+      />
```

Comment:
Ahh, I tried this earlier but I just realized it didn't work because I put `{...props}` on the last line instead of the first. This is better I think.

Reply:
oops, I included a gap that shouldn't be there. But yes!